addons/mail/static/src/models/rtc_session.js

```diff
@@ -192,8 +177,8 @@ registerModel({
             if (!this.messaging) {
                 return;
             }
-            return (this.partner && this.messaging.currentPartner === this.partner) ||
-                (this.guest && this.messaging.currentGuest === this.guest);
+            return (this.channelMember.persona.partner && this.messaging.currentPartner === this.channelMember.persona.partner) ||
```

Comment:
I've got a crash `this.channelMember` undefined when ending a call

Reply:
It's strange though because `channelMember` is required and not a compute.

Are there possibly other methods from python where a channel member is sent but without it's full data?

Reply:
It looks like yes in a lot of cases, it's probably better to make `channelMember` not required and handle the case when it is not set?

Reply:
Note that code before diff had check on existence of partner that probably was hiding the problem.